src/discord.ts

```diff
@@ -17,6 +24,58 @@ discord.on("ready", async () => {
   )) as TextChannel;
 });
 
+discord.on("messageReactionAdd", async (messageReaction: MessageReaction, user: User | PartialUser) => {
+  if (user.bot) {
+    return;
+  }
+
+  if (messageReaction.partial) {
+    await messageReaction.fetch();
+  }
+
+  const { guild } = messageReaction.message;
+
+  if (!guild) {
+    return;
+  }
+
+  const reactionRole: DiscordReactionRoles | undefined = config.discord.reactionRoles.find(
```

Comment:
I'm not sure if `undefined` is allowed here.

Reply:
undefined here is possible. find returns the first match or undefined if there are no matches.